src/NServiceBus.Transport.RabbitMQ/Configuration/AmqpConnectionString.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace NServiceBus.Transport.RabbitMQ
+{
+    class AmqpConnectionString
+    {
+        public static Action<RabbitMQTransport> Parse(string connectionString)
```

Comment:
This is parsing a connection string. The idea that this is returning an entire `RabbitMQTransport` instance to do that seems like a major smell to me.

We used to have a `ConnectionConfiguration` class for a reason.

Reply:
It does not return the instance but rather an action that configures that instance. Plus the `RabbitMQTransport` is now a configuration object only.